### ISSUE TITLE
Use `mocha-jsdom`?

### ISSUE THREAD

**Issue by benmosher** [danvk/mocha-react#7]
I found this package: https://www.npmjs.com/package/mocha-jsdom

...after reading your blog post. It was as simple to set up as:

``` javascript
var jsdom = require('mocha-jsdom');

describe('...', function () {
  jsdom();
  it('...', function () {
    // normal React TestUtils.renderIntoDocument, etc. goes here
  });
});
```

The function you suggest gave me errors I did not understand during the require of `jsdom` proper. This did not. Very straightforward.

For your consideration. 😎

**Comment by danvk** [comment]
Cool! Want to send me a PR to update the examples?

**Comment by benmosher** [comment]
Created PR #8.

**Comment by danvk** [comment]
Closed via #8.
